server.go

```diff
@@ -53,6 +65,8 @@ func (server *ToDoServer) DeleteTask(w http.ResponseWriter, req *http.Request) {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
 	}
-	server.ToDoStore.DeleteTask(target)
+	server.ToDoStore.DeleteTask(req.Context(), target)
 	fmt.Fprintf(w, "Task id %d deleted\n", target)
+	server.Statsd.Count("todo_tasks_deleted.count", int64(server.ToDoStore.Deleted), []string{"environment:dev"}, 1)
```

Comment:
If you were to submit a count with value 1 to represent the 1 task that just got deleted, then this would be a good metric.
However since you're submitting with a value of `server.ToDoStore.Deleted`, this will repeat values that always include the previous value. This is how openmetrics counters work and this is called a "monotonic count". 
Alternatively, this could be submitted as a gauge.

It really depends on what is most useful to track for this HTTP server.
Do we want to have an accurate idea of how many total tasks have been deleted over time?
Or do we want to see a value that tracks how many tasks were deleted within the given time window?

(hopefully this makes sense, please feel free to ping me directly and we can discuss)